src/main/java/Date.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * Formats date for output.
+ */
+public class Date {
```

Comment:
It is great that you created a custom Date and Timing class to format your date printing, but perhaps you would have had your work cut out for you if you had used the Java Date library instead! They have methods to print your dates and times in different formats too.